<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchWindow</class>
 <widget class="QMainWindow" name="BatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>400</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QTableWidget" name="libraryWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Source (Hidden)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Library</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">Status (hidden)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">Full path (hidden)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Filename</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artist tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Album tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comment tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Grouping tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Key tag</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Detected key</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Ready</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runBatchButton">
        <property name="text">
         <string>Run Batch Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBatchButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
